src/feedback_plugin/data_processing/charts.py

```diff
@@ -42,6 +54,56 @@ def compute_server_count_by_month(start_date: datetime,
     }
 
 
+def compute_feature_count_by_month(start_date: datetime,
+                                   end_date: datetime,
+                                   start_closed_interval: bool,
+                                   feature: str,
+) -> dict[str, dict[str, list[str] | list[int]]]:
+    uploads = get_uploads(start_date, end_date, start_closed_interval)
+
+    feature_query = """
```

Comment:
I never saw this construct before. Pretty cool using RawSQL.

I do want to check the performance of this query vs a query without a subselect. I'll get back to you with this.

Reply:
Well, I have never used Django before so it was new for me too. I didn't check EXPLAIN output or anything, so it might not be optimal.